package.json

```diff
@@ -157,7 +162,7 @@
     "watch": "yarn start -d watch,start core:start --watchTheme ",
     "build": "grunt build",
     "test": "grunt test",
-    "tslint": "tslint -c tslint.json --project tsconfig.json",
+    "tslint": "eslint public/app public/e2e-test public/test public/vendor --ext=.js,.ts,.tsx",
```

Comment:
don't think we need to lint public/vendor, old vendored libs that will have hundreds of errors